packages/form-js-viewer/src/render/components/form-fields/Textfield.js

```diff
@@ -19,12 +19,19 @@ export function Textfield(props) {
 
   const { required } = validate;
 
-  const [onInputChange, flushOnChange] = useFlushDebounce(({ target }) => {
+  const [onChange, flushOnChange] = useFlushDebounce(({ target }) => {
     props.onChange({
       value: target.value,
     });
   });
 
+  /**
+   * @param {import('preact').JSX.TargetedEvent<HTMLInputElement, Event>} event
+   */
+  const onInputChange = (event) => {
```

Comment:
I couldn't really think of a good way to test this that wouldn't be too cumbersome since the errors happens with shadow dom + react